cassdegrees/static/js/main.js

```diff
@@ -16,12 +16,12 @@ function checkKeys(event) {
  * Makes the web page go back to the homepage
  */
 function goBack() {
-    window.location.href= "/";
+    window.location.href= "/admin/home/";
```

Comment:
Can we use relative links here maybe to prevent this from being a problem in the future?

Reply:
I have done it slightly differently, take a look when I push up my changes.